WordPress/WordPressTest/ViewRelated/Post/Views/PostCardStatusViewModelTests.swift

```diff
@@ -111,6 +111,26 @@ class PostCardStatusViewModelTests: XCTestCase {
         expect(viewModel.status).to(equal(i18n("We'll publish the post when your device is back online.")))
         expect(viewModel.statusColor).to(equal(.warning))
     }
+
+    func testVersionConflictStatusMessage() {
```

Comment:
I think we can break those tests into 3 different tests. Also, you can add breaklines to make it easy to spot the [Arrange-Act-Assert](https://wiki.c2.com/?ArrangeActAssert) parts.

One thing that I did on the tests above was to add a comment explaining what the test does. I find it easier to read the comment instead of the function name. Anyway, the comment itself isn't a pattern adopted over the project, so up to you. :)